docs/Protocol/Specifications/Epoch.md

```diff
@@ -8,6 +8,8 @@ An epoch denotes a range of block heights on the data source blockchain where sm
 
 All peers synchronize and achieve consensus around incremental epochs according to their specific responsibilities. For example, snapshotter peers reach consensus around snapshots—base and aggregate—per epoch. This information gets finalized into the protocol state as snapshot datasets available as content identifiers on decentralized storage mediums like IPFS and Filecoin.
 
+![Epoch](/images/Epoch.png)
```

Comment:
the diagrams width needs to be shortened to be closer to a 1:1 ratio. Right now the stretched out, landscape ratio diagram is scaled down on the docs and barely visible

Reply:
Image updated